yarn-project/archiver/src/archiver/archiver_store_test_suite.ts

```diff
@@ -536,36 +524,6 @@ export function describeArchiverDataStore(
           ],
         ]);
       });
-
-      it('is not possible to add public logs by tag if they are invalid', async () => {
```

Comment:
This test case is now deprecated because the logs were invalid due to there being an expectations of logs being prefixed with log length and this expectation is no longer there (so the logs became valid and the test failed).
![image](https://github.com/user-attachments/assets/a50164eb-ca69-4ec3-941f-7368002c9b94)